src/commands/info/releasenotes/display.ts

```diff
@@ -53,13 +50,14 @@ export default class Display extends SfCommand<DisplayOutput | undefined> {
   };
 
   public async run(): Promise<DisplayOutput | undefined> {
+    const HIDE_NOTES = this.config.bin === 'sf' ? 'SF_HIDE_RELEASE_NOTES' : 'SFDX_HIDE_RELEASE_NOTES';
+    const HIDE_FOOTER = this.config.bin === 'sf' ? 'SF_HIDE_RELEASE_NOTES_FOOTER' : 'SFDX_HIDE_RELEASE_NOTES_FOOTER';
```

Comment:
I wonder if we should make this work like the others, so that it checks both `SF_` and `SFDX_`. This is how the hooks in `sf v2` work https://github.com/salesforcecli/cli/blob/v2/src/hooks/display-release-notes.ts#L25

What do you think?